sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -57,7 +57,7 @@ export type Answers = string;
 
 // @public
 export type AsciiFoldingTokenFilter = BaseTokenFilter & {
-    odatatype: "#Microsoft.Azure.Search.AsciiFoldingTokenFilter";
+    odataType: "#Microsoft.Azure.Search.AsciiFoldingTokenFilter";
```

Comment:
this is a breaking change, isn't it?

Reply:
It is. I figured I didn't want to touch the types that Autorest generated, but I can go ahead and manually revert the renames. Alternatively, I can bump the major version.

Reply:
Per offline this can be resolved with a Swagger transform https://github.com/Azure/autorest/blob/02bdb739e32076f1fe413c897a3d778b130b9bfc/packages/libs/configuration/resources/directives.md

Reply:
It turns out that backticks are the escape character for Swagger transforms. The necessary transforms were already there and just needed to be escaped.